client/app/components/routes/access-requests/form/UserOperatorReview.tsx

```diff
@@ -48,6 +81,8 @@ export default function UserOperatorReview({
       isStatusPending={userOperator.status === Status.PENDING}
       onApprove={approveRequest}
       onReject={rejectRequest}
+      onRequestChange={requestChange}
```

Comment:
technically there isn't any mention in the ticket or wireframes about Request Changes on a prime admin access request. I suggest confirming with either @nanyangpro or @hannavovk that we want to add the feature for requesting changes to prime admin requests. (Addn'l context: this PR also introduces Request Changes feature on new Operations and Operators, which we do want).

Reply:
> technically there isn't any mention in the ticket or wireframes about Request Changes on a prime admin access request. I suggest confirming with either @nanyangpro or @hannavovk that we want to add the feature for requesting changes to prime admin requests. (Addn'l context: this PR also introduces Request Changes feature on new Operations and Operators, which we do want).

Oh damn nice catch, I think I am confused by some of this business process. The ticket did say `Pending Operator or Operation`, where do I add that for the `Operator` then if this isn't it?

Reply:
so at the moment you're actually blocked. Brianna has a PR in progress (just waiting on some touch-ups) that separates out the approval of an Operator and a prime admin request into 2 separate review processes on the front-end.

My question for Hanna/Nan is whether we want to allow Change Requests on prime admin requests - if not, we'll want to remove that enum from the UserOperator model, and hide the button when reviewing prime admin.

Reply:
Awesome thanks for clarifying that. Should be an easy update either way, I will bring it up at standup tomorrow.